src/launcher/Launcher.cc

```diff
@@ -47,6 +49,8 @@ void Launcher::show() {
 	populateChoices();
 	initializeInputsFromDefaults();
 
+	const Fl_PNG_Image icon("logo32.png", logo32_png, 1374);
```

Comment:
Should this rather be `sizeof(logo32_png)`?

Reply:
I don't think it really matters. It is supposed to be an upper limit to guard against corruption.